node_modules/col-canvas/lib/poller.js

```diff
@@ -626,106 +626,133 @@ var handleSubmission = function(ctx, users, assignment, activities, submissionAs
     'objectType': CollabosphereConstants.ACTIVITY.OBJECT_TYPES.CANVAS_SUBMISSION,
     'metadata': {
       'submission_id': submission.id,
-      'attempt': submission.attempt
+      'attempt': submission.attempt,
+      'sync_files': assignment.category.visible
```

Comment:
@paulkerschen Nit-picky comment: If 'sync_files' is a boolean I would expect the key name to be an adjective (or similar), not a noun. E.g., 'requires_sync'. Am I understanding its purpose?